src/registrar/assets/js/get-gov.js

```diff
@@ -230,4 +230,36 @@ function handleValidationClick(e) {
 })();
 
 
+/**
+ * An IIFE that attaches a click handler for our dynamic nameservers form
+ *
+ * Only does something on a single page, but it should be fast enough to run
+ * it everywhere.
+ */
+(function prepareForms() {
+  let serverForm = document.querySelectorAll(".server-form")
+  let container = document.querySelector("#form-container")
+  let addButton = document.querySelector("#add-form")
+  let totalForms = document.querySelector("#id_form-TOTAL_FORMS")
+
+  let formNum = serverForm.length-1
+  addButton.addEventListener('click', addForm)
+
+  function addForm(e){
+      e.preventDefault()
```

Comment:
Now that the button is type="button", do we need to preventDefault?

Reply:
Likely true. I can test it manually locally. (We don't have a Javascript-capable testing setup yet.)